modules/customizing-the-home-page/proc-customizing-the-home-page-cards.adoc

```diff
@@ -56,6 +64,7 @@ dynamicPlugins:
 
 Search::
 +
+You can use the *SearchBar* card to provide essential search functionality directly on the Home page. 
```

Comment:
To clarify that this is optional, and for consistency, WDYT of this (for all occurences):

```suggestion
Optionally, use the *SearchBar* card to provide essential search functionality directly on the Home page. 
```

Reply:
Additionally, for readability, consider transforming these sentences into ordered list items, rather than a description list:

1. Optionally, configure the *SearchBar* card to provide essential search functionality directly on the Home page.  (+ example)
2. Optionally, configure the *QuickAccessCard* card to function as a customizable shortcut panel.
3. etc.